composer.json

```diff
@@ -30,7 +30,8 @@
         "guzzle/guzzle": "~3.7",
         "ornicar/akismet-bundle": "*@dev",
         "ornicar/gravatar-bundle": "dev-master@dev",
-        "antimattr/google-bundle": "dev-master@dev"
+        "antimattr/google-bundle": "dev-master@dev",
```

Comment:
Apparantly, animattr tagged a couple of versions in the mean time :grinning: https://packagist.org/packages/antimattr/google-bundle, can we use a stable version?